templates/email/default/archive-old-enquiries.html

```diff
@@ -26,11 +26,17 @@ <h1 style="[% h1_style %]">Your reports on [% site_name %]</h1>
     which we've listed below.
   </p>
   <p style="[% p_style %]">
-    All of your reports will have been received and reviewed by Shropshire, so if
+    All of your reports will have been received and reviewed by [% council_name %], so if
     your report is no longer an issue, you don't need to do anything.
   </p>
   <p style="[% p_style %]">
-    If you believe that the issue has not been resolved you can <a href="https://improvingyourroads.shropshire.gov.uk/">report it again here.</a>
+    If you believe that the issue has not been resolved you can <a href="
+    [% IF council_url %]
```

Comment:
`[%~` here would mean get rid of the whitespace between the opening HTML quote and this opening template tag (and could do same throughout this if statement).